[Docs] Clarify a route for checking stories for appropriateness (#1232)
exercism/v3: docs/concept-exercises.md

- Start with a third-level heading starting with a number (.e.g `### 1.`, `### 2.`)
- Describe which function/method the student needs to define/implement (e.g. `Implement method X(...)` that takes an A and returns a Z),
- Provide an example usage of that function. These examples should be different examples to those given in the tests.

> - We place high value on making Exercism’s content safe for everyone and so often err on the side of caution in deciding whether stories are appropriate or not. If you are unsure if something might work or not, please mention @exercism/leadership and we can clarify.

Reviewer: What about:

```suggestion
- The contents must adhere to Exercism's [Code of Conduct](https://exercism.io/code-of-conduct), as we place high value on making Exercism’s content safe and inclusive for everyone. Feel free to mention @exercism/leadership if you have any questions or would like some help.
```

Author: I don’t feel like this has anything to do with CoC though. The story about weighing yourself wasn’t in any way a violation of the CoC and it wasn’t offensive in any way. But for some people it could be triggering. 

The point here really is that this is Exercism's content, rather than a person's interactions within Exercism. So we're going to police the stories etc.

Reviewer: Right. I still feel that we're making for too big a deal of this :)

Reviewer: I'd still like to shorten it. Maybe add something about us looking at it together?

```suggestion
- We place high value on making Exercism’s content safe for everyone. If you'd like to check a story with us, please mention @exercism/leadership and we'll look at it together.
```

By the way, the heading is "Each task must conform to the following standard:", which does not lead nicely into "We place high value on making" as this is not just about the tasks, but also about the story part. As such, maybe we should make it not a bullet point but a separate bit of text.

- Describe which function/method the student needs to define/implement (e.g. `Implement method X(...)` that takes an A and returns a Z),
- Provide an example usage of that function. These examples should be different examples to those given in the tests.

> We place high value on making Exercism’s content safe for everyone and so often err on the side of caution in deciding whether stories are appropriate or not. While we are careful about what we merge, we appreciate that it's hard to be aware of what may be seen as problematic, so we'll always assume you're acting in good faith and do our best to catch any issues in review in a non-confrontational way. If you'd like to check a story with us, please mention @exercism/leadership and we'll look at it together. Here are some guiding points:

Reviewer 2: ```suggestion
We place high value on making Exercism's content safe for everyone and so often err on the side of caution in deciding whether stories are appropriate or not. While we are careful about what we merge, we appreciate that it's hard to be aware of what may be seen as problematic, so we'll always assume you're acting in good faith and do our best to catch any issues in review in a non-confrontational way. If you'd like to check a story with us, please mention @exercism/leadership and we'll look at it together. Here are some guiding points:
```

(there was a unicode `'`)

Author: > (there was a unicode ')

I just read that twice before I noticed that comment!

> - Try to make sure the story can be understood and welcoming by everyone. If the story contains in-jokes or regional slang, try to think of alternative phrases.

Reviewer 2: "can be understood and welcoming by everyone" sounds off to me because of the "by everyone"

Author: You wrote that.

:)

Author: ```suggestion
- Try to make sure the story is welcoming and can be understood by everyone. If the story contains in-jokes or regional slang, try to think of alternative phrases.
```

Reviewer 2: Hence the "They likely need to be phrased differently" in my comment ;P 

That sounds better, thanks
